sys-apps/smcipmitool/smcipmitool-2.19.0.171201.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eapi7-ver java-vm-2
+
+MY_DATE="$(ver_cut 4)"
+MY_PN="SMCIPMITool"
+MY_PV="$(ver_cut 1-3)"
+
+DESCRIPTION="An out-of-band utility for interfacing with SuperBlade and IPMI devices via CLI"
+HOMEPAGE="https://www.supermicro.com/"
+SRC_URI="amd64? ( ftp://ftp.supermicro.com/utility/${MY_PN}/${MY_PN}_${MY_PV}_build.${MY_DATE}_bundleJRE_Linux_x64.tar.gz )
+	x86? ( ftp://ftp.supermicro.com/utility/${MY_PN}/${MY_PN}_${MY_PV}_build.${MY_DATE}_bundleJRE_Linux.tar.gz )"
+
+LICENSE="supermicro"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="bundled-jre bundled-stunnel"
+
+RDEPEND="sys-devel/gcc
+	sys-libs/glibc
+	sys-libs/ncurses:5
+	!bundled-jre? ( virtual/jre )
+	!bundled-stunnel? ( net-misc/stunnel )"
+
+RESTRICT="bindist fetch mirror strip"
+
+S="${WORKDIR}"
+
+QA_PREBUILT="opt/smcipmitool/libiKVM*.so opt/smcipmitool/libjcurses*.so /opt/smcipmitool/libSharedLibrary*.so opt/smcipmitool/BMCSecurity/linux/stunnel32 opt/smcipmitool/BMCSecurity/linux/stunnel64 /opt/smcipmitool/jre/*"
+
+pkg_nofetch() {
+	elog "Please download ${A} from"
+	elog "https://www.supermicro.com/SwDownload/UserInfo.aspx?sw=0&cat=IPMI"
+	elog "and place it in your DISTDIR directory."
+}
+
+src_install() {
+	# Choose ARCH
+	if use amd64; then
+		local my_arch="${MY_PN}_${MY_PV}_build.${MY_DATE}_bundleJRE_Linux_x64"
+	else
+		local my_arch="${MY_PN}_${MY_PV}_build.${MY_DATE}_bundleJRE_Linux"
+	fi
+
+	# Install binary
+	exeinto /opt/smcipmitool
+	doexe ${my_arch}/SMCIPMITool
+
+	# Install libs
+	exeinto /opt/smcipmitool
+	doexe ${my_arch}/DrvRedirNative.dll
```

Comment:
Do we actually need the DLLs on Linux?

Reply:
To be honest, I don't know. It's works without DLLs here, but I can't test every IPMI function on my supermicro board. So I choosed to include the for "just in case". Remove it?

Reply:
Your choice. I don't have a strong opinion.

Reply:
Maybe check via `file` if they're Windows or just weird-named Linux files.

Reply:
Good hint, thanks. Acording to file, there are all windows dll files.
So I am dropping them all, as it works for me without them.

DrvRedirNative.dll:  PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
iKVM32.dll:          PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
iKVM64.dll:          PE32+ executable (DLL) (GUI) x86-64, for MS Windows
libjcurses64.dll:    PE32+ executable (DLL) (console) x86-64, for MS Windows
libjcurses.dll:      PE32 executable (DLL) (console) Intel 80386 (stripped to external PDB), for MS Windows
SharedLibrary32.dll: PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
SharedLibrary64.dll: PE32+ executable (DLL) (GUI) x86-64, for MS Windows